### DISCUSSION TITLE
What are the "default" themes?

### DISCUSSION THREAD

**Discussion by betweenvenus** [primer/github-vscode-theme#181] [Q&A]
Just looking for a little clarity! What exactly are the "default" themes? They look different than the standard versions of the themes obviously, but what does the naming convention mean? Default in what regard? I notice that Dark Default seems to be a much closer match to github.com's new dark mode.

**Comment by simurai** [comment]
> I notice that Dark Default seems to be a much closer match to github.com's new dark mode.

Yes, that's basically it. If you go to https://github.com/settings/appearance, the themes are called "Default light", "Default dark" and "Dark dimmed":

![image](https://user-images.githubusercontent.com/378023/120268062-1ffbf180-c2e0-11eb-937f-7126e37480e6.png)

In VS Code it's the same except that "default" is at the end. The additional "GitHub Light" and "GitHub Dark" themes are older version that users preferred to keep around. See https://github.com/primer/github-vscode-theme/discussions/114.

**Reply by betweenvenus** [reply]
Thank you for clearing that up!

**Reply by ascarter** [reply]
It would help to add `Legacy` to the title of the old themes to differentiate more clearly.
